sdk/src/main/java/com/silanis/esl/sdk/examples/ConditionalFieldUpdateExample.java

```diff
@@ -0,0 +1,87 @@
+    private final  FieldId fieldId1 = new FieldId("fieldId1");
+    private final  FieldId fieldId2 = new FieldId("fieldId2");
+
+    public DocumentPackage retrievedPackageWithoutConditions;
+    public DocumentPackage retrievedPackageWithConditions;
+
+    public static void main(String... args) {
+        new ConditionalFieldUpdateExample().run();
+    }
+
+    @Override
+    protected void execute() {
+
+        DocumentPackage superDuperPackage = PackageBuilder.newPackageNamed(getPackageName())
+                .describedAs("Description")
+                .withSigner(SignerBuilder.newSignerWithEmail(email1)
+                        .withFirstName("Patty")
+                        .withLastName("Galant"))
+                .withDocument(newDocumentWithName("Document")
+                        .withId(documentId)
+                        .fromStream(documentInputStream1, DocumentType.PDF)
+                        .withSignature(signatureFor(email1)
+                                .withId(signatureId)
+                                .onPage(0)
+                                .atPosition(400, 100)
+                                .withField(textField()
+                                        .withId(fieldId1)
+                                        .onPage(0)
+                                        .atPosition(0, 0))
+                                .withField(checkBox()
+                                        .withId(fieldId2)
+                                        .onPage(0)
+                                        .atPosition(0, 0))))
+                .build();
+
+        PackageId packageId = eslClient.createPackageOneStep(superDuperPackage);
+        retrievedPackage = eslClient.getPackage(packageId);
+
+        //Update field and add condition to it
+        FieldCondition condition = new FieldCondition();
+        condition.setId("ConditionId");
+        condition.setCondition("document['DocumentId'].field['fieldId2'].value == 'X'");
+        condition.setAction("document['DocumentId'].field['fieldId1'].enabled = true");
+        List<FieldCondition> conditions = new ArrayList<FieldCondition>();
+        conditions.add(condition);
+
+        ConditionalField conditionalField = new ConditionalField();
+        conditionalField.setId(fieldId2);
+        conditionalField.setConditions(conditions);
+        conditionalField.setStyle(FieldStyle.UNBOUND_CHECK_BOX);
+
+        eslClient.getApprovalService().updateConditionalField(packageId, documentId, signatureId, conditionalField);
+        retrievedPackageWithConditions = eslClient.getPackage(packageId);
+
+        //Update field and remove condition from it
+        conditions.clear();
+        conditionalField.setConditions(conditions);
+        eslClient.getApprovalService().updateConditionalField(packageId, documentId, signatureId, conditionalField);
+        retrievedPackageWithoutConditions = eslClient.getPackage(packageId);
+
+    }
+}
```

Comment:
There is no reason to have ConditionalFieldUpdateExample and CreatePackageWithConditionalFieldExample separately. Please unify into one Example.
And you can simply name the Example just like ConditionalFieldExample.